bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/handler/LivisiDeviceHandler.java

```diff
@@ -0,0 +1,1022 @@
+            commandSetDimLevel(command, bridgeHandler);
+        } else if (CHANNEL_ROLLERSHUTTER.equals(channelUID.getId())) {
+            commandRollerShutter(command, bridgeHandler);
+        } else if (CHANNEL_TARGET_TEMPERATURE.equals(channelUID.getId())) {
+            commandUpdatePointTemperature(command, bridgeHandler);
+        } else if (CHANNEL_OPERATION_MODE.equals(channelUID.getId())) {
+            commandSetOperationMode(command, bridgeHandler);
+        } else if (CHANNEL_ALARM.equals(channelUID.getId())) {
+            commandSwitchAlarm(command, bridgeHandler);
+        } else {
+            logger.debug("UNSUPPORTED channel {} for device {}.", channelUID.getId(), deviceId);
+        }
+    }
+
+    private void commandSwitchDevice(Command command, LivisiBridgeHandler bridgeHandler) {
+        if (command instanceof OnOffType) {
+            bridgeHandler.commandSwitchDevice(deviceId, OnOffType.ON.equals(command));
+        }
+    }
+
+    private void commandSetDimLevel(Command command, LivisiBridgeHandler bridgeHandler) {
+        if (command instanceof DecimalType) {
+            final DecimalType dimLevel = (DecimalType) command;
+            bridgeHandler.commandSetDimLevel(deviceId, dimLevel.intValue());
+        } else if (command instanceof OnOffType) {
+            if (OnOffType.ON.equals(command)) {
+                bridgeHandler.commandSetDimLevel(deviceId, 100);
+            } else {
+                bridgeHandler.commandSetDimLevel(deviceId, 0);
+            }
+        }
+    }
+
+    private void commandRollerShutter(Command command, LivisiBridgeHandler bridgeHandler) {
+        if (command instanceof DecimalType) {
+            final DecimalType rollerShutterLevel = (DecimalType) command;
+            bridgeHandler.commandSetRollerShutterLevel(deviceId,
+                    invertRollerShutterValueIfConfigured(rollerShutterLevel.intValue()));
+        } else if (command instanceof OnOffType) {
+            if (OnOffType.ON.equals(command)) {
+                bridgeHandler.commandSetRollerShutterStop(deviceId, ShutterActionType.DOWN);
+            } else {
+                bridgeHandler.commandSetRollerShutterStop(deviceId, ShutterActionType.UP);
+            }
+        } else if (command instanceof UpDownType) {
+            if (UpDownType.DOWN.equals(command)) {
+                bridgeHandler.commandSetRollerShutterStop(deviceId, ShutterActionType.DOWN);
+            } else {
+                bridgeHandler.commandSetRollerShutterStop(deviceId, ShutterActionType.UP);
+            }
+        } else if (command instanceof StopMoveType) {
+            if (StopMoveType.STOP.equals(command)) {
+                bridgeHandler.commandSetRollerShutterStop(deviceId, ShutterActionType.STOP);
+            }
+        }
+    }
+
+    private void commandUpdatePointTemperature(Command command, LivisiBridgeHandler bridgeHandler) {
+        if (command instanceof QuantityType) {
```

Comment:
I was reminded by other ongoing PR review that `DecimalType` should also be handled (and assumed to be Celsius). See for example:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBaseThingHandler.java#L481